unyt/tests/test_array_functions.py

```diff
@@ -590,14 +589,9 @@ def test_matrix_transpose(namespace):
 @pytest.mark.skipif(
     NUMPY_VERSION < Version("2.0.0dev0"), reason="vecdot is new in numpy 2.0"
 )
-@pytest.mark.parametrize("namespace", [None, "linalg"])
-def test_vecdot(namespace):
-    if namespace is None:
-        func = np.vecdot
-    else:
-        func = getattr(np, namespace).vecdot
+def test_vecdot():
```

Comment:
I guess this test should be renamed `test_linalg_vecdot` and moved closer to its relatives `test_linalg_*`

Reply:
Made the name change and updated the skipif. As to moving it, its current neighbours are `test_linalg_outer, test_linalg_cross, test_linalg_matmul, test_linalg_matrix_norm, test_matrix_transpose, test_linag_vecdot, test_linalg_vector_norm, test_linalg_svd, test_linalg_svdvals`, so there doesn't seem to be a strong case to move it.

Reply:
oh, I missed that it was *already* where it should. Sorry !